pyanaconda/modules/timezone/timezone.py

```diff
@@ -216,3 +220,31 @@ def install_with_tasks(self):
                 ntp_servers=self.time_sources
             )
         ]
+
+    def start_geolocation(self):
+        """Start geolocation, if not already started"""
+        if self._geoloc_task is not None:
+            # geolocation has already has started
+            return
+
+        self._geoloc_task = GeolocationTask()
+        self._geoloc_task.start()
+
+    @property
+    def geolocation_result(self):
```

Comment:
I think this should be just a passive property that should return whatever is in `_geoloc_result`. The task should handle the updates.